tests/OSPSuite.DataBinding.Tests/ElementBinderValidatorSpecs.cs

```diff
@@ -22,7 +23,8 @@ protected override void Context()
          A.CallTo(() => _elementToValidate.Source).Returns(_source);
          A.CallTo(() => _elementToValidate.PropertyName).Returns("FirstName");
          A.CallTo(() => _elementToValidate.GetValueFromControl()).Returns("Toto");
-         _elementToValidate.ParentBinder = _screenBinder;
+         A.CallTo(() => _elementToValidate.ParentBinder).Returns(_screenBinder);
+         A.CallTo(() => _elementToValidate.Control).Returns(new Label());
```

Comment:
That was the line needed to eliminate stack overflow exception. Looks like it was related to tests, not to the code itself.

Reply:
Appveyor still not running any tests though